Makefile

```diff
@@ -157,12 +157,20 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+.PHONY: markdownlint
+markdownlint: ## Check formatting of documentation sources
+	@markdownlint-cli2 "Documentation/**/**.md" "#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
+
 .PHONY: yamllint
 yamllint:
 	yamllint -c .yamllint deploy/examples/ --no-warnings
 
+.PHONY: golangci-lint
+golangci-lint:
+	@golangci-lint run
```

Comment:
We can actually run golangci-lint with `go` pretty easily without requiring users to preinstall it.

```
go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
```

Reply:
great. Thanks for the hint! I wasn't aware. I am incorporating it.